compiler/rustc_const_eval/src/interpret/memory.rs

```diff
@@ -877,12 +877,7 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
         if let Some(fn_val) = self.get_fn_alloc(id) {
             let align = match fn_val {
                 FnVal::Instance(instance) => {
-                    // Function alignment can be set globally with the `-Zmin-function-alignment=<n>` flag;
-                    // the alignment from a `#[repr(align(<n>))]` is used if it specifies a higher alignment.
-                    let fn_align = self.tcx.codegen_fn_attrs(instance.def_id()).alignment;
-                    let global_align = self.tcx.sess.opts.unstable_opts.min_function_alignment;
```

Comment:
Is there any way we can have a lint against accessing `tcx.sess.opts` in the interpreter?  Or is there some way the query machinery could help us ensure this? This is a soundness bug waiting to happen. :/  And it can't be just the interpreter, right? Other queries also must be cross-crate-stable in the same way?